graphql_test.go

```diff
@@ -216,6 +240,158 @@ func TestHelloWorld(t *testing.T) {
 	})
 }
 
+func TestHelloWorldStructFieldResolver(t *testing.T) {
```

Comment:
this doesn't call `t.Parallel()` on purpose?

Reply:
oops. Oversight. Added to this and the other new test in this PR.